performant-software/mel-website
# Convert all content to Jekyll

## What does this PR do?
This PR will merge the final part for the Jekyll conversion:
- Extracted CSS, JS
- Extracted Jekyll Template(s), includes, etc
- Extracted Posts from pages
- Keep design same
- All pages were created to match the exact permalink from the original website
- Closes #1 

## How to test?
- For testing porpuses, all the pages have the Jekyll version and the HTML version. 
- To see the Jekyll version navigate normally
- To see the HTML version include "-" before the page link e.g.
          /archive.html (Jekyll version)
          /-archive.html (HTML version)
- All the original pages will be removed before the site goes to production.
